### ISSUE TITLE
Github registry tagging

### ISSUE THREAD

**Issue by meeDamian** [meeDamian/simple-qemu#9]
:octocat: Registry allows for one more section in a path of a docker image.

Which is kinda amazing, because it allows for much more flexible grouping, ex:

There can be a `meeDamian/simple-qemu/arm` package that contains all versions of said image for `arm`, but there can also, and in parallel be a package `meeDamian/simple-qemu/v4.1.0` that contains all possible variants for that specific version.

**tl;dr:** take advantage of :octocat: Registry tagging scheme.

**Comment by meeDamian** [comment]
done, although gh registry seems quite buggy ATM…
